NetworkConfigurations/NetworkAddressingAndFilteringSubCategories/Stateless-NAT.md

```diff
@@ -0,0 +1,140 @@
+---
+title: Stateless Network Address Translation
+grand_parent: Network Configuration
+parent: Network Addressing and Filtering
+nav_order: 4
+layout: default
+---
+
+# Stateless Network Address Translation (NAT)
+
+Stateless Network Address Translation (NAT), also known as static NAT, establishes a permanent mapping between a private IP address and a single public address.
+
+To configure stateless NAT, the `tc iproute2` tool is utilized, along with the `tc-nat` action. This action is combined with the `flower` filter rule and `ingress` qdisc to enable static NAT entry offloading. Below is the command format:
+
+```
+tc ... flower ... action nat { ingress | egress } <OLD> <NEW>
+```
+
+Where:
+
+- `<OLD>` represents the IP address to be translated.
+- `<NEW>` represents the IP address to translate to.
+- `ingress` translates destination addresses (performs DNAT).
+- `egress` translates source addresses (performs SNAT).
+
+To implement NAT one-to-one mapping, two rules are required: one for private-to-public direction and another for public-to-private direction.
+
+### Example Configuration
```

Comment:
I see this example configuration, but I do not know if you've tested it. If you did, please add a section on how you verified this config.